handbook/engineering/2021_org.md

```diff
@@ -1,46 +1,32 @@
-    - [Extensibility Team](web/index.md)
-      - Engineering Manager: Extensions
-      - Product Manager: Extensions
-      - Frontend Engineers: 4
-    - [Code Host Integrations Team](web/index.md)
-      - Engineering Manager: Code Host Integrations
-      - Product Manager: Code Host Integrations
-      - Frontend Engineers: 4
-  - Director of Engineering: Core Services
-    - [Backend Infrastructure Team](backend-infrastructure/index.md)
-      - Engineering Manager: Backend Infrastructure
-      - Backend Engineers: 4
-      - Dev Ops Engineers: 2
-    - [Distribution Team](distribution/index.md)
-      - Engineering Manager: Distribution
-      - Product Manager: Distribution
-      - Dev Ops Engineers: 4
-      - Backend Engineers: 2
-    - [Security Team](security/index.md)
-      - Engineering Manager: Security
-      - Backend Engineers: 3
-      - Security Engineers: 3
-    - [Web Infrastructure Team](web/index.md)
-      - Engineering Manager: Web Infrastructure
-      - Frontend Engineers: 4
+  - Director of Engineering - Product
+    - [Search Core](search/index.md#search-core) (future split of current Search team)
+    - [Search Growth](search/index.md#search-growth) (future split of current Search team)
+    - [Campaigns](campaigns/index.md)
+    - [Code Intelligence](code-intelligence/index.md)
+    - [Code Insights](web/index.md#code-insights) (future team, currently owned by Web team)
+    - [Extensions and Integrations](web/index.md#extensions-and-integrations) (future team, currently owned by Web team)
+  - Director of Engineering - Infrastructure
+    - [Cloud](cloud/index.md)
+    - [Deployment and Operations Platform](distribution/index.md#deployment-and-operations-platform)
+    - [Developer Platform](distribution/index.md#developer-platform)
+    - [Security](security/index.md)
+    - [Backend Platform](backend-infrastructure/index.md) (future team, currently owned by Cloud team)
+    - [Web Platform](web/index.md#web-infrastructure) (future team, currently owned by Web team)
+
+Every team here is cross-functional, with all the skills and roles necessary to achieve the team's mission.
+
+Examples:
+
+- Product teams own their own on-call and operations, using infrastructure provided by Deployment and Operations Platform team.
+  - Is search broken/down? That is the search team's problem to solve.
+  - Is our whole Cloud deployment broken/down? That is the Cloud team's problem to solve.
+  - Is product team X lacking the tools they need to appropriately monitor their services? That is the Deployment and Operations Platform team's problem to solve.
+- Product teams own the security of their parts of the application. They should engage and consult with security experts on our team to get ahead of security concerns.
+- To a first approximation, each team, including infrastructure teams, need product managers. Each team has customers (for infrastructure teams, customers are internal). Most teams need designers too.
```

Comment:
Does this mean each designer will be considered **members of** one certain team like each PM, as opposed to only being _primarily associated_ with a certain team?

Reply:
I am not sure what the meaningful distinction is between those two options.

PMs and Designers are part of the functional team (but their manager is not the engineering manager).

In an ideal world, PMs and Designers are 1:1 with teams. In our current, and less than ideal world, we don't have enough PMs/designers so each PM/Designer might be involved with multiple teams.